qiskit/providers/models/backendconfiguration.py

```diff
@@ -756,6 +756,28 @@ def control(self, qubits: Iterable[int] = None,
             raise BackendConfigurationError("This backend - '{}' does not provide channel "
                                             "information.".format(self.backend_name))
 
+    def frames(self) -> List[Channel]:
+        """
+        Gets the frames of the backend in serializable format.
+
+        Returns:
+            frames: A List of lists. Sublist i is a list of channel names that belong
+                to frame i.
+        """
+        n_qubits = self.n_qubits
+        frames = []
+        for qubit in range(n_qubits):
+            frame_ = [DriveChannel(qubit).name]
```

Comment:
I think we can store channel object as-is here to avoid later channel-name mapping.

Reply:
That's what I though at first. However, this info goes into `run_config` and ends up in the QObj header. I find this nice as it shows how the frames were defined but means that this information must be serializable which is why the function above does `Channel.name`. Therefore we have two options:
1. Leave as is and have the info in the QObj header
2. remove the frame info from the run_config and therefore the QObj header and use channels instead of their names.

I have a slight preference for option 1. but 2. would also be fine.